realsense_ros_camera/src/base_realsense_node.cpp

```diff
@@ -951,6 +960,7 @@ void BaseRealSenseNode::publishFrame(rs2::frame f, const ros::Time& t)
     ++(_seq[stream]);
     auto& info_publisher = _info_publisher[stream];
     auto& image_publisher = _image_publishers[stream];
+
```

Comment:
try not to add extra white space changes. People upstream tend to not like them.